kubernetes/src/main/java/io/kubernetes/client/openapi/ApiClient.java

```diff
@@ -103,7 +103,7 @@ private void init() {
     json = new JSON();
 
     // Set default User-Agent.
-    setUserAgent("Kubernetes Java Client/11.0.1-SNAPSHOT");
+    setUserAgent("Kubernetes Java Client");
```

Comment:
Perhaps we should also add the Java runtime version?

Reply:
added todo line